src/index.js

```diff
@@ -439,6 +469,17 @@ class ServerlessAppsyncPlugin {
           Tags: !config.tags ? undefined : this.getTagsConfig(config),
         },
       },
+      ...(config.authenticationType === 'AWS_LAMBDA' && {
+        [`${logicalIdGraphQLApi}LambdaAuthorizerPermission`]: {
+          Type: 'AWS::Lambda::Permission',
+          Properties: {
+            Action: 'lambda:InvokeFunction',
+            FunctionName: this.getLambdaAuthorizerConfig(config).AuthorizerUri,
+            Principal: 'appsync.amazonaws.com',
+            SourceArn: { Ref: logicalIdGraphQLApi },
+          },
+        },
+      }),
```

Comment:
🔥 awesome!

Aren't we missing the same for `additionalAuthenticationProviders` though?